ledger/forest/forest.go

```diff
@@ -50,44 +51,50 @@ func (f *Forest) Save(tree *trie.MTrie, paths []ledger.Path, parent flow.StateCo
 	f.steps[commit] = s
 }
 
-// Has returns whether a state commitment matches one of the trees within the forest.
 func (f *Forest) Has(commit flow.StateCommitment) bool {
 	_, ok := f.steps[commit]
 	return ok
 }
 
-// Tree returns the matching tree for the given state commitment.
-func (f *Forest) Tree(commit flow.StateCommitment) (*trie.MTrie, bool) {
+func (f *Forest) Tree(commit flow.StateCommitment) (*trie.Trie, bool) {
 	s, ok := f.steps[commit]
 	if !ok {
 		return nil, false
 	}
+
 	return s.tree, true
 }
 
-// Paths returns the matching tree's paths for the given state commitment.
```

Comment:
No, it's because I rewrote a lot of stuff and didn't include the comments, but I will add them back now, thanks :) 👍 It's never too obvious when it comes to public functions/methods IMO!